envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/SnapshotProperties.kt

```diff
@@ -123,6 +123,7 @@ class RoutesProperties {
     var admin = AdminRouteProperties()
     var status = StatusRouteProperties()
     var authorization = AuthorizationProperties()
+    var headersToRemove: List<String> = emptyList()
```

Comment:
IMO routes properties is not the best place for this property, because we set it only on ingress routes, not egress routes.

Maybe we should create `ingress` section on the same level as `egress` section?:

```
envoy:
  snapshot:
     egress:
         ...
     ingress:
       headersToRemove:
```

Reply:
I've moved property as you suggested